### DISCUSSION TITLE
posts/Go/go-kit-tutorial-02/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#302] [Announcements]
https://www.liwenzhou.com/posts/Go/go-kit-tutorial-02/

 李文周的Blog提供免费的全套Go语言学习教程。本文主要介绍了go-kit的教程的第二篇，介绍了如何使用go-kit构建基于gRPC的微服务，文章的最后还额外介绍了如何为gRPC Server编写本地测试代码。

**Comment by mingo-liu** [comment]
在当下grpc v1.70.0版本中，把 grpc.DialContext 改为 grpc.NewClient，"bufnet" 改为 "passthrough:///bufnet"
